pkgs/games/factorio/versions.json

```diff
@@ -0,0 +1,58 @@
+{
+  "x86_64-linux": {
+    "alpha": {
```

Comment:
```suggestion
    "experimental": {
```

Reply:
The "release branch" (or "build type") is named "alpha" still, even though the game is now released - this is the actual game, with all graphics, etc. included.

The "alpha" build type/release branch has two versions - experimental, and stable - of which experimental is usually expected to eventually end up in stable once it's, well, stabilised.

Reply:
I wouldn't call it by the interal api names just to make it easier for everyone. Does not really matter.

They called it really weird on their API page (https://factorio.com/api/latest-releases) because alpha is basically the full game which makes no sense. Everything else is logical.

BTW you don't need to explain Factorio to me. I played almost as many hours as nixpkgs has open PRs. 🕹️